### PR TITLE
fix-quote-decimal-input

### PR DESCRIPTION
**Pull request by fetok12**

### CODE REVIEW

File: web/src/components/QuickTrade/index.js [hunk 1]
```diff
@@ -419,7 +419,7 @@ const QuickTrade = ({
 								onInputChange={onChangeTargetAmount}
 								forwardError={() => {}}
 								limits={side === 'buy' ? SIZE : PRICE}
-								decimal={side === 'buy' ? increment_unit : PAIR2_STATIC_SIZE}
+								decimal={side === 'buy' ? PAIR2_STATIC_SIZE : increment_unit}
```
Comment (ahslr) [inline web/src/components/QuickTrade/index.js:422]: Seems to me that this change is either off or incomplete as the **convert** input and **to** input should not hold the same logic for decimals.

**Review by ahslr — CHANGES_REQUESTED** [review]

File: web/src/components/QuickTrade/index.js (hunk 1 above)
Comment (ahslr) [reply web/src/components/QuickTrade/index.js:422]: The previous implementation added by Ram also seems off to me. It should be increment_size or increment_price based on the side of the trade. If the pair does not exist, which means it's a broker trade, we need to use a static decimal place.

Comment (abeikverdi) [reply web/src/components/QuickTrade/index.js:422]: The coins on each side of quick trade here can follow the increment_unit alternatively.

Comment (abeikverdi) [reply web/src/components/QuickTrade/index.js:422]: I am closing this PR. Amir will clean this up in the new quick trade update.

**Pull request closed without merging**